src/components/typography/typography.stories.mdx

```diff
@@ -126,6 +126,79 @@ It's also possible to create ordered and unordered lists using the `List`, `List
   </Story>
 </Preview>
 
+## Examples of white-space
```

Comment:
I'm not sure if there's any benefit from showing these stories as we don't have stories for other css rules

Reply:
That's fine. These are easily deleted 😄